CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Contributing
+
+We provide and maintain SDKs for the benefit of our developer community. Feedback, detailed bug reports, and focused PRs are appreciated. Thank you in advance!
+
+When contributing to this SDK, please:
+
+- Maintain the minimum PHP version (found under `require.php` in `composer.json`).
+- Code to the [PSR-2 standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md).
+- Write tests and run them with `composer test`.
+- Keep PRs focused and change the minimum number of lines to achieve your goal.
+
+To run tests on the SDK, you'll need to create a `.env` file in the root of this package with the following entries:
+
+- `DOMAIN` - Auth0 domain for your test tenant
+- `APP_CLIENT_ID` - Client ID for a test Regular Web Application
+- `APP_CLIENT_SECRET` - Client Secret for a test Regular Web Application
+- `NIC_ID` - Client ID for a test Non-Interactive Client Application
```

Comment:
**A note in case of a future refactor.** Since you're using the "CLIENT" word for other constants, might as well use it here and be more descriptive. In the end these are just names one should easily see and remember. e.g. `NON_INTERACTIVE_CLIENT_ID`